ui/component/claimListDiscover/index.js

```diff
@@ -13,6 +13,7 @@ import { selectFollowedTags } from 'redux/selectors/tags';
 import { selectMutedAndBlockedChannelIds } from 'redux/selectors/blocked';
```

Comment:
- [ ] Following Page needs handling for Empty case

<img width="635" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/21921473-6612-441d-aa1d-4fd0556c7051">

Reply:
- I din't add the cog to following because it rarely has an entire row of claims and it would make it look kinda weird. The cog is in the row bellow. That's the intended behavior. 
- Didn't notice the entry in the navigation :man_facepalming: